backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -39,4 +39,36 @@ const groups = pgTable('groups', {
   updatedAt: timestamp('updated_at').notNull().defaultNow(),
 });
 
-export { groups, users };
+const trucks = pgTable(
+  'trucks',
+  {
+    id: serial('id').primaryKey(),
+    createdAt: timestamp('created_at').notNull().defaultNow(),
+    updatedAt: timestamp('updated_at').notNull().defaultNow(),
+    manufacturer: varchar('manufacturer').notNull(),
+    capacity: varchar('capacity').notNull(),
+    pricePerKm: varchar('price_per_km').notNull(),
+    licensePlateNumber: varchar('license_plate_number').notNull(),
+  },
+  (trucks) => {
+    return {
+      uniqueLicensePlateNumber: uniqueIndex(
+        'trucks_license_plate_number_idx',
+      ).on(trucks.licensePlateNumber),
+    };
+  },
+);
+
+const usersTrucks = pgTable('users_trucks', {
+  id: serial('id').primaryKey(),
```

Comment:
Maybe we can use [Composite Primary Key](https://orm.drizzle.team/docs/indexes-constraints#composite-primary-key) here?